src/components/AudioRecitationCard.tsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react'
+import { SlCloudDownload } from "react-icons/sl";
+import { FaFacebookSquare } from "react-icons/fa";
+import { AiOutlineMail } from "react-icons/ai";
+import { BsTwitterX } from "react-icons/bs";
+
+
+export default function AudioRecitationCard() {
```

Comment:
the audio card component should only include te neceesaary code for a singlle instance of it, other shoukld be through mapping and props for the correct data.